src/main/java/io/github/maeves2/SnowballNebula.java

```diff
@@ -36,7 +36,7 @@
  * @since 1.0.0
  * @author MaeveS2
  */
-public class SnowballNebula extends ListenerAdapter {
+public class oh SnowballNebula extends ListenerAdapter {

```

Comment:
Is `oh` a new Java keyword I'm unaware of?

Reply:
its not even in the last commit smh